topo_processor/util/conversions.py

```diff
@@ -67,3 +67,21 @@ def quarterdate_to_date_string(value: str) -> str:
         return date_string_utc
 
     return value
+
+
+def h_i_photo_type_to_linz_geospatial_type(value: str) -> str:
```

Comment:
I would name the argument `value` like `photo_type`? or something more specific than `value`